contrib/preprocessing/extract-particles-properties-from-vtu.py

```diff
@@ -37,27 +37,14 @@
 # Diameter
 diameters = df["diameter"][:]
 
-# FEM force
-fem_f_x = df["fem_force"][:, 0]
-fem_f_y = df["fem_force"][:, 1]
-fem_f_z = df["fem_force"][:, 2]
-
-# FEM force
-fem_t_x = df["fem_torque"][:, 0]
-fem_t_y = df["fem_torque"][:, 1]
-fem_t_z = df["fem_torque"][:, 2]
-
 with open(output_file_name, 'w') as file:
     # Write content to the file
     file.write(
-        "p_x; p_y; p_z; v_x  ; v_y; v_z; w_x; w_y; w_z; diameters; fem_force_x; fem_force_y; fem_force_z; "
-        "fem_torque_x; fem_torque_y; fem_torque_z \n")
+        "p_x; p_y; p_z; v_x  ; v_y; v_z; w_x; w_y; w_z; diameters; \n")
```

Comment:
```suggestion
        "p_x; p_y; p_z; v_x ; v_y; v_z; w_x; w_y; w_z; diameters; \n")
```

Reply:
I think this would change test files, so I am not sure it is worth changing such a small typo :)